devices/b-parasite/b-parasite.json

```diff
@@ -0,0 +1,415 @@
+          "refresh.interval": 86400,
+          "read": {
+            "at": "0x0021",
+            "cl": "0x0001",
+            "ep": 10,
+            "fn": "zcl:attr"
+          },
+          "parse": {
+            "at": "0x0021",
+            "cl": "0x0001",
+            "ep": 10,
+            "eval": "Item.val = Attr.val / 2",
+            "fn": "zcl:attr"
+          }
+        },
+        {
+          "name": "config/offset",
+          "default": 0
+        },
+        {
+          "name": "config/on"
+        },
+        {
+          "name": "config/reachable"
+        },
+        {
+          "name": "state/lastupdated"
+        },
+        {
+          "name": "state/moisture",
+          "parse": {
+            "at": "0x0000",
+            "cl": "0x0408",
+            "ep": 10,
+            "eval": "Item.val = Attr.val + R.item('config/offset').val",
+            "fn": "zcl:attr"
+          },
+          "default": 0
+        }
+      ]
+    },
+    {
+      "type": "$TYPE_LIGHT_LEVEL_SENSOR",
+      "restapi": "/sensors",
+      "uuid": [
+        "$address.ext",
+        "0x0a",
+        "0x0400"
+      ],
+      "items": [
+        {
+          "name": "attr/id"
+        },
+        {
+          "name": "attr/lastannounced"
+        },
+        {
+          "name": "attr/lastseen"
+        },
+        {
+          "name": "attr/manufacturername"
+        },
+        {
+          "name": "attr/modelid"
+        },
+        {
+          "name": "attr/name"
+        },
+        {
+          "name": "attr/swversion"
+        },
+        {
+          "name": "attr/type"
+        },
+        {
+          "name": "attr/uniqueid"
+        },
+        {
+          "name": "config/battery",
+          "refresh.interval": 86400,
+          "read": {
+            "at": "0x0021",
+            "cl": "0x0001",
+            "ep": 10,
+            "fn": "zcl:attr"
+          },
+          "parse": {
+            "at": "0x0021",
+            "cl": "0x0001",
+            "ep": 10,
+            "eval": "Item.val = Attr.val / 2",
+            "fn": "zcl:attr"
+          }
+        },
+        {
+          "name": "config/on"
+        },
+        {
+          "name": "config/reachable"
+        },
+        {
+          "name": "config/tholddark",
+          "default": 12000
+        },
+        {
+          "name": "config/tholdoffset",
+          "default": 7000
+        },
+        {
+          "name": "state/dark",
+          "default": false
+        },
+        {
+          "name": "state/daylight",
+          "default": false
+        },
+        {
+          "name": "state/lastupdated"
+        },
+        {
+          "name": "state/lightlevel",
+          "parse": {
+            "at": "0x0000",
+            "cl": "0x0400",
+            "ep": 10,
+            "eval": "Item.val = Attr.val",
+            "fn": "zcl:attr"
+          },
+          "default": 0
+        },
+        {
+          "name": "state/lux",
+          "default": 0
+        }
```

Comment:
Here I'd suggest to take the very same passage from one of the Philips motion sensors. Depending where you have put your DDF locally, it might be required to copy the .js script file over and amend its path to have it tested.

Reply:
Good point! Added conversion script 4f6c41b2fb9a7c9abaaf0660a467cdd575020f42 51c893516ca5bc7d5555f9e2f2eea52711f79b66